src/cosmosdb-preview/azext_cosmosdb_preview/commands.py

```diff
@@ -107,3 +48,17 @@ def load_command_table(self, _):
         g.command('list', 'list')
         g.show_command('show', 'get')
         g.command('delete', 'begin_delete', confirmation=True, supports_no_wait=True)
+
+    with self.command_group('cosmosdb graph', cosmosdb_graph_resources_sdk, client_factory=cf_graph_resources, is_preview=True) as g:
+        g.custom_command('create', 'cli_cosmosdb_graph_create', supports_no_wait=True)
+        g.custom_command('exists', 'cli_cosmosdb_graph_exists', supports_no_wait=True)
```

Comment:
Why should `exists` command support `--no-wait`? That's kind of wired ><

Reply:
removed